cdap-docs/application-templates/source/etl/templates/transformations/projection.rst

```diff
@@ -29,3 +66,39 @@ Only simple types are supported (boolean, int, long, float, double, bytes, strin
 simple type can be converted to bytes or a string. Otherwise, a type can only be converted
 to a larger type. For example, an int can be converted to a long, but a long cannot be
 converted to an int.
+
+.. rubric:: Example 
+
+::
+
+  {
+    "name": "Projection",
+    "properties": {
+      "drop": "ts,headers",
+      "rename": "cost:price",
+      "convert": "cost:double"
+    }
+  }
+ 
+This example drops the 'ts' and 'headers' fields. It also renames the 'cost' field 
+to 'price', and changes its type to a double. For example, if the transform receives the
+input record::
```

Comment:
This example drops the 'ts' and 'headers' fields. It also changes the type of the 'cost' field 
to a double and renames it 'price'. For example, if the transform receives the
input record::